spec/factories/rankinglists.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```

Comment:
Does this really output a valid Rankinglist? This does not look like it would pass Event's validations.